test/integration/install-variants_test.js

```diff
@@ -29,7 +31,7 @@ suite('install-variants', function() {
           assert.fail(err);
           done(err);
         }
-        runCommand(binPath, ['install', '--variants', '--offline'], {cwd: tmpPath}).then(() => {
+        runCommand(binPath, ['install', '--variants'], {cwd: tmpPath}).then(() => {
```

Comment:
My bet is that Travis is setting an environment variable for the Bower cache, which is overriding our setting.